openedx2zim/scraper.py

```diff
@@ -101,6 +108,8 @@ def __init__(
         name,
         title,
         description,
+        #added long_description here
```

Comment:
Remove this useless comment

Reply:
removed all these useless comments as well